src/tests/EShopworld.WorkerProcess.UnitTests/Stores/CosmosDbLeaseStoreTests.cs

```diff
@@ -204,6 +204,110 @@ public async Task TestTryCreateLeaseAsyncConflict()
             result.Result.Should().BeFalse();
         }
 
+        [Fact, IsUnit]
+        public async Task TestAddLeaseRequestAsync_WhenAddingLeaseRequest_ShouldReturnTrue()
+        {
+            // Arrange
+            _mockDocumentClient.Setup(m => m.CreateDocumentAsync(
+                    It.Is<Uri>(u => u == UriFactory.CreateDocumentCollectionUri(_options.Database, "LeaseRequests")),
+                    It.IsAny<object>(),
+                    It.IsAny<RequestOptions>(),
+                    It.IsAny<bool>(),
+                    It.IsAny<CancellationToken>()))
+                .ReturnsAsync(new Document().ToResourceResponse(HttpStatusCode.Created));
+
+            // Act
+            var result = await _store.AddLeaseRequestAsync("worktype", 1, Guid.NewGuid());
+
+            // Assert
+            result.Should().BeTrue();
+            _mockTelemetry.Verify(tel=>tel.Publish(It.IsAny<ExceptionEvent>(),It.IsAny<string>(),
```

Comment:
When/how do you test that data passed in your call is what is being passed to the DB?

Reply:
sure! additional assertion added